File: tests/fixtures/independent/typescript/sample.ts
import { helper as helperFn } from "./utils";
import * as path from "path";

export class Service {
  run(): void {
    helperFn();
    path.basename("a/b");
  }
}

export function entry(): void {
  const service = new Service();
  service.run();
}

File: tests/fixtures/independent/java_ctor_member/Sample.java
package fixture.sample;

class Service {
  int run() {
    return 1;
  }
}

public class Sample {
  private final Service service;

  public Sample(Service service) {
    this.service = service;
  }

  public int handle() {
    return this.service.run();
  }
}
